device-types/Dell/EMC-MD1420.yml

```diff
@@ -0,0 +1,64 @@
+---
+manufacturer: Dell
+model: EMC MD1420
+slug: dell-emc-md1420
+part_number: emc-md1420
+front_image: true
+rear_image: true
+comments: |
+  [Dell EMC MD1420 Datasheet](https://i.dell.com/sites/csdocuments/Shared-Content_data-Sheets_Documents/en/DellStorage_MD1400_MD1420_082114.pdf)
+u_height: 2
+is_full_depth: true
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 600
+module-bays:
+  - name: Slot 1
+    position: '1'
+  - name: Slot 2
+    position: '2'
+  - name: Slot 3
+    position: '3'
+  - name: Slot 4
+    position: '4'
+  - name: Slot 5
+    position: '5'
+  - name: Slot 6
+    position: '6'
+  - name: Slot 7
+    position: '7'
+  - name: Slot 8
+    position: '8'
+  - name: Slot 9
+    position: '9'
+  - name: Slot 10
+    position: '10'
+  - name: Slot 11
+    position: '11'
+  - name: Slot 12
+    position: '12'
+  - name: Slot 13
+    position: '13'
+  - name: Slot 14
+    position: '14'
+  - name: Slot 15
+    position: '15'
+  - name: Slot 16
+    position: '16'
+  - name: Slot 17
+    position: '17'
+  - name: Slot 18
+    position: '18'
+  - name: Slot 19
+    position: '19'
+  - name: Slot 20
+    position: '20'
+  - name: Slot 21
+    position: '21'
+  - name: Slot 22
+    position: '22'
+  - name: Slot 23
+    position: '23'
+  - name: Slot 24
+    position: '24'
```

Comment:
Are these all for storage media? If so, Storage media is not currently supported, please remove these definitions